p2pclient/streams.go

```diff
@@ -4,15 +4,14 @@ import (
 	"bytes"
 	"encoding/binary"
 	"fmt"
-	"io"
-	"net"
-	"os"
-
 	ggio "github.com/gogo/protobuf/io"
 	"github.com/gogo/protobuf/proto"
 	pb "github.com/libp2p/go-libp2p-daemon/pb"
-	peer "github.com/libp2p/go-libp2p-peer"
+	"github.com/libp2p/go-libp2p-peer"
 	ma "github.com/multiformats/go-multiaddr"
+	"github.com/multiformats/go-multiaddr-net"
+	"io"
+	"net"
```

Comment:
we put the system imports on top on their own, and named imports again.

Reply:
Apologies, my IDE has been inserting them in the top group. Fixed.